src/main/java/org/apache/commons/lang3/time/DurationFormatUtils.java

```diff
@@ -264,7 +264,12 @@ static String format(final Token[] tokens, final long years, final long months,
             }
             if (isLiteral) {
                 if (!inOptional || !lastOutputZero) {
-                    buffer.append(value.toString());
+                    final String val = value.toString();
```

Comment:
val --> s?

Reply:
What do you mean by `val --> s?` ? Assign `val` to `s`?

Reply:
No, rename val to s since it's just a string. In this case there's no obvious difference between val and value and s is pretty standard in Java for arbitrary string variables that don't have a deeper meaning.

Reply:
The name 's' is already in use in this scope so your comment was confusing to me.

Reply:
OK, then perhaps valueString or valueAsString or something like that

Reply:
On line 249 what's the actual type? Could we just cast that to string?